pkg/placement/controller.go

```diff
@@ -237,6 +247,11 @@ func (c *Controller) run(workers int) error {
 	}
 	c.logger.Info("All caches synced")
 
+	// populate the PlacementResolver with entries for existing placements
+	if err := c.populatePlacementResolverWithExistingPlacements(); err != nil {
+		return fmt.Errorf("failed to populate the PlacementResolver for the existing placements")
```

Comment:
`err` should make it into the returned error.

Reply:
Fixed.